### PR TITLE
Lesson12

### PR DESCRIPTION
**Pull request by Naumych**
Creating tables and models:
Route<<->>Station
Train<<->Station
Train<<->Route
Ticket<<->User
Ticket<<->Train

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+# Offense count: 4
+# Cop supports --auto-correct.
+# Configuration parameters: MinSize.
+# SupportedStyles: percent, brackets
+Style/SymbolArray:
+  EnforcedStyle: brackets
+
+Rails/HasAndBelongsToMany:
+  Enabled: false
```
Comment (svidlonix) [inline .rubocop.yml]: Please add last empty line.

File: app/models/train.rb [hunk 2]
```diff
@@ -3,5 +3,7 @@
 class Train < ApplicationRecord
   validates :number, presence: true
 
-  belongs_to :railway_station
+  belongs_to :current_station, class_name: 'RailwayStation', foreign_key: :current_station_id
```
Comment (svidlonix) [inline app/models/train.rb]: there is no needs rename DB column if you add  **foreign_key** it also can be 
_belongs_to :current_station, class_name: 'RailwayStation', foreign_key: :railway_station_id
￼Write_

**Review by svidlonix — COMMENTED** [review]
A few things

File: app/models/train.rb (hunk 2 above)
Comment (Naumych) [reply app/models/train.rb]: Fixed

File: .rubocop.yml (hunk 1 above)
Comment (Naumych) [reply .rubocop.yml]: Fixed

File: app/models/train.rb (hunk 2 above)
Comment (svidlonix) [reply app/models/train.rb]: I really happy my comment is become useless.

**Pull request merged**